README.md

```diff
@@ -1,18 +1,25 @@
 # freeze-dried-raspberry-cookies
 In development
 
+yields: 30 cookies
+
 ## Ingredients
-1. 1/2 cup butter
-2. 1/4 cup powdered sugar
-3. 3/4 cup all-purpose flour
-4. 1/4 cup ground freeze dried raspberries
+1. 3/4 cup butter
+2. 3/4 cup granulated sugar
+3. 2 1/4 cup all-purpose flour
+4. 1/4 cup + 1 tbs ground freeze dried raspberries (sifted)
+6. 1 large egg
+7. 1 tsp vanilla
+8. 1/4 tsp salt
+9. 1/2 tsp baking powder
```

Comment:
Not sure if you've got a reason for it but this is a really low amount of leavening for 2+ cups of flour, which is contributing to a pretty dense cookie. For drop cookies you can go up to 1 tsp per cup of flour, maybe a little less for a chill and slice dough.

Reply:
Maybe 3/4-1 tsp then. Two cups flour to half a tsp of leaven seems fairly standard in thin, decorating-style sugar cookies but those recipes don't have added body from raspberries.